openedx/core/djangoapps/cors_csrf/decorators.py

```diff
@@ -22,6 +23,10 @@ def _inner(*args, **kwargs):  # pylint: disable=missing-docstring
             # the cross-domain version of the CSRF cookie.
             request = args[0]
             request.META['CROSS_DOMAIN_CSRF_COOKIE_USED'] = True
+            current_authenticator = getattr(request, 'successful_authenticator', None)
+            set_custom_attribute("tmp_cors_csrf_decorator_authenticator", current_authenticator)
+            set_custom_attribute('tmp_cors_csrf_decorator_referer', request.META.get('HTTP_REFERER'))
+            set_custom_attribute('tmp_cors_csrf_decorator_host', request.get_host())
```

Comment:
Thanks.
1. Using a `.` helps some observability tools better organize the attributes:
```suggestion
            set_custom_attribute("tmp_cors_csrf_decorator.authenticator", current_authenticator)
            set_custom_attribute('tmp_cors_csrf_decorator.referer', request.META.get('HTTP_REFERER'))
            set_custom_attribute('tmp_cors_csrf_decorator.host', request.get_host())
```
2. Would you mind documenting the attributes using the following docs: https://github.com/openedx/edx-django-utils/issues/328?
3. I guess this commit should be "temp: ..." (instead of "feat:"), based on your attribute names?
4. I haven't reviewed in detail, but I'm only providing general `set_custom_attribute` thoughts at this time.  Maybe either in the annotations that you add (and/or supplemented by updates to the PR description), you can explain how you envision the attributes being used to make us more confident about follow-up changes.

Reply:
Hi @robrap, thanks for your feedback. I’ve addressed those points.

Regarding point number 4, I’ve updated the description of this PR to provide a more detailed explanation.